src/decode.rs

```diff
@@ -3889,73 +3908,87 @@ static ss_size_mul: enum_map_ty!(Rav1dPixelLayout, [u8; 2]) = enum_map!(Rav1dPix
 unsafe fn setup_tile(
     c: &Rav1dContext,
     ts: &mut Rav1dTileState,
-    f: &Rav1dFrameData,
+    seq_hdr: &Rav1dSequenceHeader,
+    frame_hdr: &Rav1dFrameHeader,
+    bitdepth_max: i32,
+    sb_shift: i32,
+    f_cur: &Rav1dPicture,
```

Comment:
```suggestion
    cur: &Rav1dPicture,
```

Nit: For all of the other args that were previously inside `f`, we don't add an `f_` prefix.